btx/diagnostics/bayesian_optimization.py

```diff
@@ -88,7 +88,7 @@ def probability_of_improvement(self, X, gp_model, best_y):
     # Bayesian Optimization
 
     @classmethod
-    def run_bayesian_opt(self, config):
+    def run_bayesian_opt(cls, config):
```

Comment:
For future reference: https://peps.python.org/pep-0008/#function-and-method-arguments